css/styles.css

```diff
@@ -344,47 +344,179 @@
   min-height: 250px;
   padding: 50px 30px;
   text-align: center;
+  padding-bottom: 0;
 }
 
 .motivation p {
-  font-family: Inter, serif;
+  font-family: 'Inter', sans-serif;
   color: #172b4d;
-  font-size: 30px;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  font-weight: 700;
+}
+
+/* .contact-form {
+  height: auto;
 }
 
 .form {
   min-height: 300px;
   padding: 50px 30px;
   text-align: center;
+  padding-top: 0;
 }
 
 .form input {
-  width: 100%;
   padding: 30px 20px 10px 20px;
   border: none;
   border-bottom: 2px solid #dbd8d7;
 }
 
+.form input text {
+  width: 327px;
+  height: 18px;
+  left: 16px;
+  right: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
 .form textarea {
   margin-top: 15px;
   border: none;
   background-color: #fbf8f7;
   padding: 15px 0  0 20px;
 }
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. [You can read more about best practices for HTML & CSS here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)